uamqp/async_ops/receiver_async.py

```diff
@@ -88,7 +86,7 @@ def __init__(self, session, source, target,
                  encoding='UTF-8',
                  desired_capabilities=None,
                  loop=None):
-        self.loop = loop or get_running_loop()
+        self._internal_kwargs = get_dict_with_loop_if_needed(loop)
```

Comment:
same comment as before on the default get_running_loop() behavior for python 3.6 -> 3.9